packages/app-explorer/src/index.tsx

```diff
@@ -56,11 +56,12 @@ class ExplorerApp extends React.Component<Props, State> {
           <Tabs
             basePath={basePath}
             items={items}
+            query={true}
```

Comment:
If this is always true, do we want/need to pass it in? (With this actually we can forgo showing it on the NodeInfo page, which is probably the better experience)

Reply:
We need it because this component is also used by the other apps to show their tabs. I can try to hide the Query component when the user clicks on NodeInfo, but, precisely given its generality, the result might not be super clean.

Reply:
That is fine, you are right, probably more trouble than it is worth.